website/publish-gh-pages.sh

```diff
@@ -17,10 +17,11 @@ git checkout -- .
 git clean -dfx
 git fetch
 git rebase
-rm -Rf *
+rm -Rf $1
+mkdir ../../react-native-gh-pages/$1
 cd ../react-native/website
 node server/generate.js
-cp -R build/react-native/* ../../react-native-gh-pages/
+cp -R build/react-native/* ../../react-native-gh-pages/$1
```

Comment:
Is the website even working with this? I'm pretty sure we use absolute paths in a lot of places.

The &lt;base> element may be handy: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base